README.md

```diff
@@ -1,3 +1,3 @@
-# Northcoders News API
+# Create the environment variables
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=, with the correct database name for that environment.
```

Comment:
I would also name the databases, or say that you can find the names in the setup.sql file, just so that users don't get this step wrong, or think that any database name would work